package.json

```diff
@@ -89,7 +89,7 @@
     "semver-diff": "^2.0.0",
     "text-table": "^0.2.0",
     "throat": "^2.0.2",
-    "update-notifier": "^2.1.0",
```

Comment:
It looks like CI passed, but with this upgrading 2 major versions, there is some risk. Did you evaluate what changed in `update-notifier` between `2.1` and `4.1`? No breaking changes that affect `npm-check`?

Reply:
no I didn't

Reply:
not really familiar with the project, just need the vulnerability gone

Reply:
Just thought I'd chime in here (I'm not the submitter). I had a look at the [release notes](https://github.com/yeoman/update-notifier/releases) for the update-notifier package and, from my checking, I only see one breaking change of note: in v3.0.0 of update-notifier, support for node v6 was dropped, and they currently only support node v8 onwards. Maybe npm-check's Travis CI file needs to be updated to remove node versions 4 and 6 from it?

The other breaking changes are in v4.0.0 of update-notifier, but from my source code searches they do not affect npm-check:
- "Remove the callback option)". `callback` is a field in the options object that can be passed to `updateNotifier`. npm-check only uses the `pkg` option.
- "Rename the boxenOpts option to boxenOptions". This is one of the options you can optionally pass to notifier.notify([options]). npm-check does not use any options with that `notify` call.
- "Update boxen dependency". The release notes say that this may affect the `boxenOptions` values. npm-check does not use `boxenOpts`/`boxenOptions`.
- "Disable update notifications when NODE_ENV is test". This affects testing scenarios only so should have no impact on npm-check usage.

Reply:
@splaktar -- just bumping this thread. Thanks!